collections/business/general-notifications/event-general-notifications/payment-order-updated/event-general-notifications.json

```diff
@@ -3,7 +3,7 @@
   "id": "payment-order-updated",
```

Comment:
Please check on the README file, the branch name needs to start with `feature/*`.